cranelift/interpreter/src/interpreter.rs

```diff
@@ -191,9 +192,12 @@ pub enum HeapInit {
     FromBacking(HeapBacking),
 }
 
+pub type LibCallHandler<'a, V> = &'a dyn Fn(SmallVec<[V; 1]>) -> SmallVec<[V; 1]>;
```

Comment:
I think it might also be a good idea to return `Result<SmallVec<[V; 1]>, TrapCode>` and allow these to trap controllably.